backend/tracim_backend/lib/core/content.py

```diff
@@ -1405,7 +1406,7 @@ def _move_current(
         if new_parent and new_parent != item.parent:
             content_type = content_type_list.get_one_by_slug(item.type)
             self._check_valid_content_type_in_dir(content_type, new_parent, new_workspace)
-
+        item.owner = self._user
```

Comment:
I think `self._user` may be `None`. Please check, and if so, then the code should be `item.owner = self._user if self._user`

Reply:
this syntax is not correct , replace by : 
```
if self._user:
  item.owner = self._user
```

Reply:
done